utils/node_local_test.py

```diff
@@ -1028,6 +1029,40 @@ def il_cmd(dfuse, cmd, check_read=True, check_write=True, check_fstat=True):
     return ret
 
 
+def pil4dfs_cmd(dfuse, cmd):
```

Comment:
Does this require dfuse?  I'd have expected this to be a method on DaosServer as I've done in https://github.com/daos-stack/daos/pull/12084

Reply:
It also appears that all callers of this immediately assert that rc.returncode == 0, a safer approach here would be to have this function/method do the assert and if we need to we can add a "check=False" option here later on to disable that.